### ISSUE TITLE
Multiple monitors, multiple screens, cannot record (多显示器多屏幕无法录制问题)

### ISSUE THREAD

**Issue by 1353604736** [NickeManarin/ScreenToGif#986]
Multiple monitors, multiple screens, cannot record (多显示器多屏幕无法录制问题)

**Comment by NickeManarin** [comment]
Can you explain more the issue? 

Do you want to record something across multiple screens?

**Comment by 1353604736** [comment]
> Can you explain more the issue?
> 
> Do you want to record something across multiple screens?

Cannot record in the second screen window

**Comment by Jerrylum** [comment]
> Can you explain more the issue? 
> 
> Do you want to record something across multiple screens?

Yes, I think he has a problem about recording something across multiple screens.

> Cannot record in the second screen window
> After clicking record, there is no response to stop recording

I am not sure what he is talking about.

---

Jerry：你可以使用中文描述你所遇到的问题，我会把它翻译成英文。

Jerry: You are free to describe your issue in Chinese and I will translate it into English.

**Comment by NickeManarin** [comment]
Thanks for the help, @Jerrylum 

-------

@1353604736 Can you show me which screen recorder UI are you using? Older or newer?

![image](https://user-images.githubusercontent.com/14798947/134906954-b95c1b99-95f7-49ac-afbd-b26d7e6e37e3.png)

**Comment by 1353604736** [comment]
> 感谢您的帮助，@Jerrylum
> 
> @1353604736你能告诉我你使用的是哪个屏幕录像机用户界面吗？旧的还是新的？
> 
> ![图片](https://user-images.githubusercontent.com/14798947/134906954-b95c1b99-95f7-49ac-afbd-b26d7e6e37e3.png)

新的，旧的，都是一样的效果，录制开始，结束后，没有任何反应，结束并没有跳转到编辑界面。

**Comment by Jerrylum** [comment]
@NickeManarin 

1353604736: Both older and newer UI have the same problem. There is no response after I stopped the recording, and the application does not enter the editing interface.

**Comment by fz0000** [comment]
试一下全新安装。卸载后删除所有配置文件和临时文件，再重新安装最新版。我之前也是副屏录制出来不正常，删掉所有文件全新安装就行了。
Clean reinstall instead of upgrade might help (i.e. delete all config and temp files and reinstall with default settings). It solved my recording problem on the secondary screen.

**Comment by 1353604736** [comment]
在选择DirectX模式录制时，录制第二屏幕时，就回出现此问题
![搜狗截图20211112102253](https://user-images.githubusercontent.com/12869582/141399080-7d30ebca-d3cf-4bc9-b998-e43fd6730e99.png)

**Comment by 1353604736** [comment]
各种插件都已下载
![搜狗截图20211112103452](https://user-images.githubusercontent.com/12869582/141399174-21ce3c99-a711-4ad8-bd94-6b737504aa12.png)

**Comment by NickeManarin** [comment]
The DirectX capture method was updated in v2.35. Can you try again @1353604736 ?
If the error still persists, can you send me the error log?
It's inside that ScreenToGif > Log folder in your screenshot.

**Comment by 1353604736** [comment]
> DirectX 捕获方法已在 v2.35 中更新。你能再试一次@1353604736吗？ 如果错误仍然存​​在，您能否将错误日志发送给我？ 它位于屏幕截图中的 ScreenToGif > Log 文件夹内。

v2.35正常了，问题已解决O(∩_∩)O
